rules/unicorn.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  rules: {
+    'unicorn/catch-error-name': 'error',
+    'unicorn/custom-error-definition': 'off',
+    'unicorn/error-message': 'error',
+    'unicorn/escape-case': 'error',
+    'unicorn/explicit-length-check': ['error', { 'non-zero': 'greater-than' }],
+    'unicorn/filename-case': ['error', { case: 'camelCase' }],
+    'unicorn/import-index': 'error',
+    'unicorn/new-for-builtins': 'error',
+    'unicorn/no-abusive-eslint-disable': 'error',
+    'unicorn/no-array-instanceof': 'error',
+    'unicorn/no-console-spaces': 'error',
+    'unicorn/no-fn-reference-in-iterator': 'off',
+    'unicorn/no-for-loop': 'error',
+    'unicorn/no-hex-escape': 'error',
+    'unicorn/no-new-buffer': 'error',
+    'unicorn/no-process-exit': 'off',
+    'unicorn/no-unreadable-array-destructuring': 'off',
+    'unicorn/no-unsafe-regex': 'error',
+    'unicorn/no-unused-properties': 'off',
+    'unicorn/no-zero-fractions': 'error',
+    'unicorn/number-literal-case': 'error',
+    'unicorn/prefer-add-event-listener': 'error',
+    'unicorn/prefer-event-key': 'error',
+    'unicorn/prefer-exponentiation-operator': 'error',
+    'unicorn/prefer-flat-map': 'error',
+    'unicorn/prefer-includes': 'error',
+    'unicorn/prefer-node-append': 'error',
+    'unicorn/prefer-node-remove': 'error',
+    'unicorn/prefer-query-selector': 'error',
+    // Prefer `Array.from` for performance benefits.
+    'unicorn/prefer-spread': 'off',
+    'unicorn/prefer-starts-ends-with': 'error',
+    'unicorn/prefer-text-content': 'error',
+    // Type checks are performed by TypeScript compiler.
+    'unicorn/prefer-type-error': 'off',
+    'unicorn/prefer-text-content': 'error',
+    'unicorn/prevent-abbreviations': [
+      'error',
+      { checkDefaultAndNamespaceImports: true, whitelist: ['ref', 'doc'] },
```

Comment:
I'm not sure about this one. This rule has an autofixer. If we'd like to remove whitelisting for `ref` and `doc`, it would automatically change it to `Reference` and `Document`. I think it might be a wise thing to do, since we started using `ref` and `doc` because of Firestore SDK, but new contributors might not know what do these stand for. I'd personally remove whitelisting for these two, but I wanted to discuss it with more people first. What's your take on this?

Reply:
Removed as agreed offline.